test/Project.toml

```diff
@@ -2,6 +2,7 @@
 CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 CUDAKernels = "72cfdca4-0801-4ab0-bf6a-d52aa10adc57"
+CloudMicrophysics = "6a9e3e04-43cd-43ba-94b9-e8782df3c71b"
```

Comment:
I'm not sure this should be added, since it changes how tests can be performed.

I recently learned about [https://github.com/JuliaTesting/TestEnv.jl](TestEnv.jl), maybe this can help with running your local tests interactively?